internal/gitops/gitops.go

```diff
@@ -281,16 +290,30 @@ func CreatePullRequest(repoIdentifier, baseBranch, featureBranch, title, body, g
 	repoOwner := strings.Split(repoIdentifier, "/")[0]
 	repoName := strings.Split(repoIdentifier, "/")[1]
 
-	authToken := github_token
+	authToken := githubToken
 	ctx := context.Background()
 	ts := oauth2.StaticTokenSource(
 		&oauth2.Token{AccessToken: authToken},
 	)
-	tc := oauth2.NewClient(ctx, ts)
+
+	//tc := oauth2.NewClient(ctx, ts)
+
+	ts2 := &oauth2.Transport{
+		Source: oauth2.ReuseTokenSource(nil, ts),
+		Base: &netHttp.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		},
+	}
+
+	tc := &netHttp.Client{
+		Transport: ts2,
+	}
 
 	client := github.NewClient(tc)
 
-	draft := true
+	draft := false
```

Comment:
should we take this input from user?

Reply:
We can reserve this interaction for later.